elastic/integrations
# Complete Linux Integration

## What does this PR do?

This completes the Linux integration, and adds the `linux` metrics from metricbeat, in addition to the linux-only metrics in `system` as well.
 
## Checklist



## How to test this PR locally

- Pull down the PR, build the registry and start the stack.
- Install the Linux integration from the store.
- Run an agent on a linux host, make sure all datasets are reporting.

## Related issues

This PR is being hampered a tad by this: https://github.com/elastic/package-spec/issues/51

We need to have two somewhat awkawrd config sets (see example) in order to get metrics from two metricbeat modules.
